src/PKSim.Presentation/Presenters/Formulations/FormulationSettingsPresenter.cs

```diff
@@ -152,9 +154,9 @@ private void adjustParameterVisibility()
          if (!Formulation.IsParticleDissolution)
             return;
 
-         var cache = new Cache<string, IParameter>(p => p.Name);
+         var cache = new Cache<string, IParameter>(p => p.Name, x => null);
          cache.AddRange(_formulationDTO.Parameters);
-         cache.Each(p => p.Visible = true);
```

Comment:
Hidding showing is not enough. We also need to update the IsDefault flag

Reply:
It's correct but a bit hard to understand what happens here.
It would be easier for understanding to show/hide parameters first and then for all parameters set 
`p.IsDefault=!p.Visible`

Reply:
You can keep it as is, not a big deal. Might consider adding some comments explaining what's going on.
Maybe I have a bad day, but I had to read it 3 times to get it.

Reply:
No no I'll update. Nothing like the moment!-)